src/pages/Compare/index.tsx

```diff
@@ -138,10 +160,23 @@ export class CompareBase extends React.Component<Props> {
   }
 
   render() {
-    const { addonId, compareInfo, match, path, version } = this.props;
+    const {
+      addonId,
+      compareInfo,
+      match,
+      path,
+      version,
+      versionFile,
+    } = this.props;
 
     const { baseVersionId, headVersionId } = match.params;
 
+    let getCodeLineAnchor;
+    if (compareInfo && compareInfo.diff) {
+      const map = new ForwardComparisonMap(compareInfo.diff);
+      getCodeLineAnchor = map.createCodeLineAnchorGetter();
+    }
+
     // TODO: set showFileInfo=true when we can:
     // https://github.com/mozilla/addons-code-manager/issues/647
```

Comment:
This patch should allow us to fix this TODO, no? If so, can you leave a comment in the linked issue?

Reply:
No, that's separate issue. It's about showing the file information (sha hash, etc)

Reply:
Well, yeah... I thought that it was blocked because no version file was passed to `VersionFileViewer`, which is solved in this patch.

Reply:
oh, I see! I'll defer it to https://github.com/mozilla/addons-code-manager/issues/647 so that it can get its own QA. We may need to change how we display the file name to make it obvious that it's the "head" file but I'm not sure.